tests/fixtures/api-fixtures.ts

```
@@ -0,0 +1,59 @@
+        await requestContext.dispose();
+      }
+    },
+    { scope: 'worker' }
+  ],
+  
+  apiClient: async ({ apiRequestContext, apiConfig }, use) => {
+    const client = new ApiClient(apiRequestContext, apiConfig.baseUrl);
+    await use(client);
+  },
+  
+  logger: async ({}, use) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)